Split Updating Guide (#2233)
theforeman/foreman-documentation: guides/common/modules/con_updating-project-to-next-minor-version.adoc

[id="updating-project-to-next-minor-version_{context}"]
= Updating {Project} to the Next Minor Version

Use this guide to update your {ProjectServer} and {SmartProxyServer} to a new minor release version, such as from {ProjectVersion}.0 to {ProjectVersion}.1.

The minor releases patch security vulnerabilities and minor issues discovered after code is released, and are often fast and non-disruptive to your operating environment.

.Prerequisite
* Back up your {ProjectServer} and all {SmartProxyServers}.

> For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.

Reviewer: I suggest distributing the prerequisite into all sections instead of having it here.

Author: Added to all the three sessions +1

Author: Added to all the three sections :+1:

> Before updating, back up your {ProjectServer} and all {SmartProxyServers}.
> For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.

Reviewer: ```suggestion
* Back up your {ProjectServer} and all {SmartProxyServers}.
For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
```

[id="updating-project-to-next-minor-version_{context}"]
= Updating {Project} to the Next Minor Version

> Use this guide to update your {ProjectServer} and {SmartProxyServer} to a new minor release version, such as from {ProjectVersion}.0 to {ProjectVersion}.1.

Reviewer: ```suggestion
You can update your {ProjectServer} and {SmartProxyServer} to a new minor release version, such as from {ProjectVersion}.0 to {ProjectVersion}.1, using the {Project} installer.
```

> 
> The minor releases patch security vulnerabilities and minor issues discovered after code is released, and are often fast and non-disruptive to your operating environment.

Reviewer: ```suggestion
The minor releases are non-disruptive to your operating environment and often fast.

{Team} recommends performing updates regularly, because the minor releases patch security vulnerabilities and minor issues discovered after code is released.
```
@ehelms @ekohl Can we go ahead with such a recommendation?

Reviewer 2: I am not sure I agree with `The minor releases are non-disruptive to your operating environment and often fast.` Mostly the first part, given the latter is covered by "often".

Reviewer: @ehelms I'm looking for a motivation for the user to update at all and how often they should. Who could advise us on this?

Author: Addressed :+1:

[id="updating-project-to-next-minor-version_{context}"]
= Updating {Project} to the Next Minor Version

> You can update your {ProjectServer} and {SmartProxyServer} to a new minor release version, such as from {ProjectVersion}.0 to {ProjectVersion}.1, using the {Project} installer.

Reviewer: ```suggestion
You can update your {ProjectServer} and {SmartProxyServer} to a new minor release version, such as from {ProjectVersion}.0 to {ProjectVersion}.1, using the {Project} maintain tool.
```
